src/mach/mod.rs

```diff
@@ -114,6 +115,37 @@ impl<'a> MachO<'a> {
             symbols::SymbolIterator::default()
         }
     }
+    pub fn compact_unwind_info(&self) -> error::Result<compact_unwind_info::CompactUnwindInfoIter> {
+        let get_unwind_info_section = || {
+            let section_name = "unwind_info";
+            for segment in &self.segments {
+                for section in segment {
+                    if let Ok((header, data)) = section {
+                        if let Ok(sec) = header.name() {
+                            if sec.len() >= 2 && &sec[2..] == section_name {
```

Comment:
why `&sec[2..]` is there some ascii prefix it is stripping here? If it's always the same, maybe it's better to have `section_name` contain that prefix and remove this part?

Reply:
That's the two underscores, for example `"__unwind_info"`. This function is structured similarly to `object` and `symbolic` which allow to search in both ELF and MachO for a section name regardless of the specific prefix.

Reply:
(yea this is a straight-up copy-paste of `symbolic` to resolve the section by name, since that functionality seemed to be absent.)